CHANGELOG.md

```diff
@@ -5,16 +5,18 @@ Full documentation for hipCUB is available at [https://rocm.docs.amd.com/project
 ## (Unreleased) hipCUB-x.x.x for ROCm 6.4.0
```

Comment:
We need to set the hipCUB version.  Also this PR deletes the extended tests entry.  For the sake of expediency we'll merge in this PR, and fix the above two things in another PR immediately.